plugin/src/main/java/io/jenkins/plugins/casc/core/HudsonPrivateSecurityRealmConfigurator.java

```diff
@@ -33,13 +41,32 @@ public HudsonPrivateSecurityRealmConfigurator() {
                     .collect(Collectors.toList()))
             .setter((target, value) -> {
                 for (UserWithPassword user : value) {
-                    target.createAccount(user.id, user.password);
```

Comment:
I'd suggest moving the for each loop to a method, allowing you to use method reference instead of lambda :smile:

Creating better readability and reducing indentation.

Reply:
Made a bunch of fixes to this class, including ensuring proper typing was used. Thanks for the feedback.